frontend/src/app/app.module.ts

```diff
@@ -11,35 +11,35 @@ import { CommonModule } from '@angular/common';
 import { HTTP_INTERCEPTORS, HttpClientModule } from '@angular/common/http';
 import { NgModule } from '@angular/core';
```

Comment:
The "active session" card can be optimized a bit: 
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/249c38f4-7749-4b40-8d62-e202c3dbc55f)

The status bar ("Session started") is a bit too wide, The title ("Persistent workspace session") should be larger and bold.